RocketChat/Rocket.Chat.ReactNative
# Chore/migrate lib user preferences to ts

## Proposed changes


Migrate user preferences to TS
And fix default browser bug on change selected browser

## Issue(s)	


## How to test or reproduce


go to:
Settings -> Default browser
Change the browser
Close and open the app
See the correct browser selected 

## Screenshots

## Types of changes




## Checklist



## Further comments
